go.mod

```diff
@@ -64,7 +64,7 @@ require (
 	github.com/go-openapi/validate v0.19.2
 	github.com/go-ozzo/ozzo-validation v3.5.0+incompatible // indirect
 	github.com/godbus/dbus v4.1.0+incompatible
-	github.com/gogo/protobuf v1.0.0
+	github.com/gogo/protobuf v0.0.0-20190410021324-65acae22fc9
```

Comment:
was there a tag that included this change? when possible, we're trying to stick to tagged versions of dependencies to be sure we don't pick up half-working releases

Reply:
Can we make that a follow up?  If we hit an issue I’d rather tag in gogo once we’re certain this works.

Reply:
Follow up == before 1.16

Reply:
sure, added back to https://github.com/kubernetes/kubernetes/issues/79234

Reply:
I used the hash of the merge commit for the PR that we wanted.
@jmarais, Should we expect that to be a "half-working" release? Do you know when is the next release that will include the reverse marshaling? Thanks!

Reply:
@smarterclayton @apelisse looks like this survived for 3 days or so. Wanna created a tag in gogo/protobuf please?

Reply:
We don't own that repo, I'll follow-up with the owner